content/ASP-NET-intro/reading/create-asp-project/index.md

```diff
@@ -0,0 +1,109 @@
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+ASP.NET is a framework in the .NET Core family that is used to build web applications. While [ASP.NET](https://learn.microsoft.com/en-us/aspnet/core/?view=aspnetcore-6.0) can be used to build a wide variety of web applications, we will be focusing on using it to build MVC web applications.
+
+## Getting Started
+
+To create a new ASP.NET MVC project, start a new project in Visual Studio.
+
+
+### Windows Users
+
+1. Use the Get started Menu to Create a new project.
+
+1. When selecting the type of project, select *ASP.NET Core Web App (Model-View-Controller)*.
+
+   There are 2 ways to find this easily:
+      1. Use the search bar.
+
+      1. Select “Web” from the dropdown menu.
+
+   Once you have your project type, click *Next*.
+
+1. Name your project `HelloASPDotNET` and put it in the appropriate directory for all of your classwork. Hit *Next*.
+
+1. Select the Framework. We are going to use .NET Core 6.0. You do not need to adjust any other options at this point. Select *Create*!
+
+1. Visual Studio creates a fully-functional web application for you.
+
+{{% notice green "Tip" "rocket" %}} 
+
+**Troubleshooting:**
+This [tutorial for Windows](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-6.0&tabs=visual-studio) can help you if you are stuck.
+
+{{% /notice %}}
+
+### Mac Users
+1. Open Visual Studio for Mac and select  *New*
+
+1. The type of project is a **Web Application (Model-View-Controller)**. In the left-side menu, select *App* from under the *Web and Console* list. Continue on to the next screen.
+
+1. Target Framework will be `.NET 6.0`. You don’t need to adjust any other settings at this time.
+
+1. Name your project `HelloASPDotNET` and put it in the appropriate directory for all of your classwork. Hit *Next*.
+
+1. Visual Studio creates a fully-functional web application for you.
+
+{{% notice green "Tip" "rocket" %}} 
+
+**Troubleshooting:**
+Microsoft created a tutorial for creating an MVC in Visual Studio, but for the Mac version they recommend the following [guide](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-7.0&tabs=visual-studio-mac).
+
+{{% /notice %}}
+
+{{% notice orange "Warning" "rocket" %}} 
+The guide is working with `.NET 7.0`. Make you that you are using `.NET 6.0` as the process is identical.
+ 
+{{% /notice %}}
+
+### All Users
+
+1. Now launch the application!
+
+   1. **Mac Users:** Click *Run*.   
+
+   1. **Windows Users:** Select `HelloAspDotNet` (or whatever you named your project) and try launching the application if it doesn't work initially.
+
+1. Eventually, your browser will open and display your application. Take note of the port number in the address bar.
```

Comment:
I missed this line the first time, except when I got to the CYU question. Found this confusing - why am I taking note?